march_hardware/include/march_hardware/IMotionCube.h

```diff
@@ -99,12 +99,20 @@ class IMotionCube : public Slave
 
   void mapMisoPDOs();
   void mapMosiPDOs();
-  void writeInitialSettings(uint8_t cycle_time);
+  bool writeInitialSettings(uint8_t cycle_time);
   /**
    * Calculates checksum on .sw file passed in string format in sw_string_ by simple summation until next empty line.
    * Start_address and end_address are filled in the method and used for downloading the .sw file to the drive.
    */
   uint32_t computeSWCheckSum(int& start_address, int& end_address);
+  /**
+   * Compares the checksum of the .sw file and the setup on the drive. If both are equal 1 is returned.
+   */
+  int verifySetup();
```

Comment:
Why not a boolean?

Reply:
I wanted to use the return of this function just like the SDO commands as a means to verify if the code has succesfully written everything away. The return values of those functions is an int and I thought it would only make sense to do the same for my method.

Reply:
I believe those methods return a working counter? I think it would be neater to either do that too (by returning the outcome of IMotionCube.cpp, line 448) or return a bool

Reply:
Okay, if you both think this would be neater I'll change these return values.